lib/core/form/components/form-base.component.ts

```diff
@@ -123,7 +123,7 @@ export abstract class FormBaseComponent {
 
         if (outcome) {
             if (outcome.name === FormOutcomeModel.SAVE_ACTION) {
-                return this.disableSaveButton ? false : this.form.isValid;
+                return !this.disableSaveButton;
```

Comment:
so it will be possible to save invalid form?

Reply:
That's right